packages/alfa-formatter/src/formatter.ts

```diff
@@ -8,7 +8,7 @@ export type Formatter<I = unknown, T = unknown, Q = never, S = T> = (
   input: I,
   rules: Iterable<Rule<I, T, Q, S>>,
   outcomes: Iterable<Outcome<I, T, Q, S>>
-) => string;
+) => Promise<string>;
```

Comment:
Due to `jsonld.compact` being async and used in the EARL formatter, we now need formatters to be async 🙈
Since formatters are only used in the CLI tool which already loads them asynchronously, this is not a real problem…